### PR TITLE
add unattended heartbeat possibility

### PR DESCRIPTION
**Pull request by kinvaris**

### CODE REVIEW

File: ovs/lib/setup.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def setup_node(ip=None, force_type=None):
                 known_passwords = json.loads(config.get('setup', 'passwords'))
             if config.has_option('setup', 'external_etcd'):
                 external_etcd = json.loads(config.get('setup', 'external_etcd'))
-            enable_heartbeats = False
+            enable_heartbeats = config.getboolean('setup', 'enable_heartbeats')
```
Comment (khenderick) [inline ovs/lib/setup.py]: Can you make it optional and set the default to `False`?

Comment (wimpers) [inline ovs/lib/setup.py]: I would prefer to have the default set to true. See discussion of the BAM in OVS-3948

Comment (khenderick) [inline ovs/lib/setup.py]: Ok. but please make it optional, as we have to update all our QA environments otherwise.

File: ovs/lib/setup.py [hunk 2]
```diff
@@ -124,7 +124,10 @@ def setup_node(ip=None, force_type=None):
                 known_passwords = json.loads(config.get('setup', 'passwords'))
             if config.has_option('setup', 'external_etcd'):
                 external_etcd = json.loads(config.get('setup', 'external_etcd'))
-            enable_heartbeats = False
+            try:
```
Comment (khenderick) [inline ovs/lib/setup.py]: Try again with a `config.has_option('setup', 'enable_heartbeats')` ;).

**Comment by kvanhijf** [comment]
And try again with default True apparently (in the 'else' stack)   :)

**Comment by kinvaris** [comment]
It should be False default right, because you would need to rewrite the QA tests?

**Comment by kinvaris** [comment]
I've set the default to True, this means that you will have to inform QA that they would have to rewrite their tests if you do not want to receive heartbeats in support portal

**Pull request merged**